tests/hikari/impl/test_entity_factory.py

```diff
@@ -2584,7 +2590,7 @@ def message_payload(
         self, user_payload, member_payload, custom_emoji_payload, partial_application_payload, embed_payload
     ):
         del member_payload["user"]
-        return {
+        payload = {
```

Comment:
What's the reasoning behind this change? 👀

Reply:
Forgot to revert this. My initial fix to add `referenced_message` was to have it reference to the same payload, lol